### DISCUSSION TITLE
<Add new features to Lists>

### DISCUSSION THREAD

**Discussion by Stein283036** [community/community#45505] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I want to add some extra info to my stared repo in different list, so I cound know which repo is of more importance to me.

**Comment by Melaniejmcclure94** [comment]
Read more information on it and the  answer is within the content this way you are likely make a more precise answer and a better choice on your question.

**Reply by Melaniejmcclure94** [reply]
https://github.com/community/community/discussions/45505#discussioncomment-4801501

**Reply by Melaniejmcclure94** [reply]
Ok did this answer help you, I need you to send me feedback
